utils/envvars/parser.go

```diff
@@ -218,10 +219,18 @@ func parseSAASHost(v string) (string, error) {
 		logrus.Infof("Using default SaaS host %q.", defaultSaaSHost)
 		return defaultSaaSHost, nil
 	}
-	if strings.Contains(v, ":") {
-		return "", errors.New("saas host can't have port set: it's signed for domain without the port")
+	if strings.HasPrefix(v, ":") {
```

Comment:
What is the point to accept `host:port` format if we throw away `port` part?

Reply:
Because it's everywhere in tests with ports. Previously we failed to start pmm-managed in those scenarios.

Reply:
In that case, we should modify our tests and use strict validation here. It may be really confusing for someone it the future when you set port but it's ignored.

Can you please create ticket for that?